src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -769,20 +769,81 @@ class ContentSidebar extends PureComponent<Props, State> {
         });
     };
 
+    /**
+     * Creates a task assignment object to be inserted into a newly created task
+     *
+
+     * @param {Function} successCallback - Task create success callback
+     * @param {Function} errorCallback - Task create error callback
+     * @return {TaskAssignment}
+     */
+    createTaskAssignmentObject(data: any) {
+        const { id: assignmentId, assigned_to, message, resolution_state } = data;
```

Comment:
why rename id?

Reply:
It was previously in a method that already had `id` defined. Fixed